sdcm/results_analyze/__init__.py

```diff
@@ -169,6 +178,18 @@ def __init__(self, es_index, es_doc_type, send_email, email_recipients, logger=N
             logger=logger,
             events=events)
 
+    def get_starred_events(self):
+        last_events = dict()
+        events_summary = dict()
+
+        if not self._events:
+            return [last_events, events_summary]
+
+        last_events['STARRED'] = self._events.get("STARRED", [])
+        events_summary['STARRED'] = len(last_events['STARRED'])
+
+        return [last_events, events_summary]
```

Comment:
What about adding all this stuff to `get_events()` instead?

Reply:
in my case, the idea is to have it separated from the other events

Reply:
They are separated by being in "STARRED" already, aren't they?

Reply:
done